components/whautomate/actions/create-contact/create-contact.mjs

```diff
@@ -0,0 +1,80 @@
+import { STAGE_OPTIONS } from "../../common/constants.mjs";
+import { parseObject } from "../../common/utils.mjs";
+import whautomate from "../../whautomate.app.mjs";
+
+export default {
+  key: "whautomate-create-contact",
+  name: "Create Contact",
+  description: "Create a new contact associated with a WhatsApp number. [See the documentation](https://help.whautomate.com/product-guides/whautomate-rest-api/contacts#/v1-contacts-1)",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    whautomate,
+    name: {
+      type: "string",
+      label: "Name",
+      description: "The name of the contact",
+    },
+    phoneNumber: {
+      type: "string",
+      label: "Phone Number",
+      description: "The WhatsApp phone number of the contact",
```

Comment:
Does `phoneNumber` require the country code? I'd put an example in the description to show the user the expected format. Example: `+15555555555`, or Example: `15555555555`.